File: src/Middleware/integrations/ordercloud.integrations.library/modelhelpers/ListFilterValue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ordercloud.integrations.library
{
    public class ListFilterValue
    {
        public string Term { get; set; } = "";
        public ListFilterOperator Operator { get; set; } = ListFilterOperator.Equal;
        public IList<int> WildcardPositions { get; set; } = new List<int>();
        public bool HasWildcard => WildcardPositions.Any();

    }
}

File: src/UI/SDK/src/models/ListFilterValue.ts
export interface ListFilterValue {
    Term?: string
    Operator?: 'Equal' | 'GreaterThan' | 'LessThan' | 'GreaterThanOrEqual' | 'LessThanOrEqual' | 'NotEqual'
    WildcardPositions?: number[]
    HasWildcard?: boolean
}